features/fields.feature

```diff
@@ -0,0 +1,228 @@
+    | burgerservicenummer | 999995078                       |
+    | fields              | burgerservicenummer             |
+    Dan heeft de persoon met burgerservicenummer '999995078' de volgende gegevens
+    | naam                | waarde    |
+    | burgerservicenummer | 999995078 |
+
+  Scenario: Gevraagde velden zijn 'root' gegevens van de persoon
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 999995078                       |
+    | fields              | burgerservicenummer,aNummer     |
+    Dan heeft de persoon met burgerservicenummer '999995078' de volgende gegevens
+    | naam                | waarde    |
+    | burgerservicenummer | 999995078 |
+    | aNummer             | 123456789 |
+
+  Scenario: Gevraagde veld is een gegeven van een gegevensgroep van de persoon
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                                       |
+    | type                | RaadpleegMetBurgerservicenummer              |
+    | burgerservicenummer | 999995078                                    |
+    | fields              | burgerservicenummer,geslachtsaanduiding.code |
+    Dan heeft de persoon met burgerservicenummer '999995078' de volgende gegevens
+    | naam                     | waarde    |
+    | burgerservicenummer      | 999995078 |
+    | geslachtsaanduiding.code | M         |
+
+  Scenario: Gevraagde veld is een gegevensgroep van de persoon
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                                  |
+    | type                | RaadpleegMetBurgerservicenummer         |
+    | burgerservicenummer | 999995078                               |
+    | fields              | burgerservicenummer,geslachtsaanduiding |
+    Dan heeft de persoon met burgerservicenummer '999995078' de volgende gegevens
+    | naam                             | waarde    |
+    | burgerservicenummer              | 999995078 |
+    | geslachtsaanduiding.code         | M         |
+    | geslachtsaanduiding.omschrijving | man       |
+
+  Scenario: Gevraagde velden zijn gegevens van één gegevengroep van de persoon
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                                                |
+    | type                | RaadpleegMetBurgerservicenummer                       |
+    | burgerservicenummer | 999995078                                             |
+    | fields              | burgerservicenummer,naam.geslachtsnaam,naam.voornamen |
+    Dan heeft de persoon met burgerservicenummer '999995078' de volgende gegevens
+    | naam                | waarde    |
+    | burgerservicenummer | 999995078 |
+    En heeft de persoon met burgerservicenummer '999995078' de volgende 'naam' gegevens
```

Comment:
dit testgeval was nog interessanter geweest wanneer in Gegeven nog meer naamgegevens zouden zitten, en dus alleen deze twee worden geleverd

Reply:
voorvoegsel is toegevoegd aan de naamgegevens